lisp/ess-r-package.el

```diff
@@ -363,6 +364,12 @@ With prefix argument ARG, run tests on current file only."
 })
 ")
 
+(defun ess-r-package-save-buffers ()
```

Comment:
Should this be `ess-project-save-buffers`? This seems generic enough. We should eventually have an ESS specific notion of projects, with `ess-project-current` picking up `.ess` files, RStudio project file, R packages, etc. This function would save buffers for projects thusly defined.

Reply:
Sounds good to me. Renamed.